src/commands.ts

```diff
@@ -640,6 +644,22 @@ export const insertFootnote = (
     ).pop()
 
     if (footnoteElement) {
+      if (isDeleted(footnoteElement.node)) {
```

Comment:
do we need to check if its the last footnote node?

Reply:
no, just if the container footnote element is exist but deleted

Reply:
okay